generators/app/utility.js

```diff
@@ -283,6 +283,11 @@ function validateServicePrincipalKey(input) {
    return validateRequired(input, `You must provide a Service Principal Key`);
 }
 
+function validateConfigUpdate(input){
+   let valid = input.toLowerCase() === 'yes' ? input : '';
+   return validateRequired(valid, 'This file must be updated with the correct config file before you select "Yes" ');
```

Comment:
is there a unified place where all of these strings go?

Reply:
This is the format for all the other "validate...". It looks like all of the strings are just passed in like this. They are all just used as return messages for the validateRequired method

Reply:
Yup, then it makes sense to follow their pattern